scripts/release.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/sh
+# Copyright (c) 2024 Hemi Labs, Inc.
+# Use of this source code is governed by the MIT License,
+# which can be found in the LICENSE file.
+
+set -e
+
+#
+# hemilabs/heminetwork release script
+#
+# Tags the latest remote commit in the heminetwork repository's main branch and
+# pushes the tag to the repository, triggering the release CI which will build
+# all necessary binaries, Docker images, etc. and create a GitHub release.
+#
+# This script will create a new remote called 'upstream' (configurable with
+# REMOTE_NAME). This remote is used to fetch the latest commit on the main
+# branch, allowing a tag to be created on the latest remote commit while
+# ignoring any local changes.
+#
+# The provided version (and all version bumps) must follow the Semantic
+# Versioning 2.0.0 specification (<major>.<minor>.<patch>), as per
+# https://semver.org/. This allows the versioning to stay consistent while
+# easily communicating the type of change the version contains to users.
+#
+# This script will always confirm with you before creating or pushing the tag,
+# in an attempt to prevent accidental tag creation or pushes.
+#
+# Usage:
+# 	release.sh <version>
+#
+# Requirements:
+#	- git (configured correctly)
```

Comment:
Some weird spacing things here.

Reply:
That's odd, my editor must have done something strange. I'll put 3 spaces instead of tabs.